src/test/java/step2/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,49 @@
+package step2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("문자열 덧셈 계산기를 통한 TDD/리팩토링 실습")
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(result).isZero();
```
도 사용할 수 있을 것 같습니다 : )